app/controllers/catalog_controller.rb

```diff
@@ -8,7 +8,7 @@ class CatalogController < ApplicationController
   include Blacklight::Searchable
   include Browse
   include ReportIssue
-  include SearchContext
+  include ::ReportIssue
```

Comment:
Is there a difference between `ReportIssue` and `::ReportIssue`?

Reply:
I really have no idea

Reply:
looks like this was changed in this PR https://github.com/psu-libraries/psulib_blacklight/pull/855/files I've fixed this to only have the one `include` statement